src/main/java/uk/gov/hmcts/juror/api/moj/report/standard/PanelMembersStatusReport.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.juror.api.moj.report.standard;
+
+import com.querydsl.core.Tuple;
+import com.querydsl.jpa.impl.JPAQuery;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.StandardReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.domain.trial.QPanel;
+import uk.gov.hmcts.juror.api.moj.report.AbstractReport;
+import uk.gov.hmcts.juror.api.moj.report.AbstractStandardReport;
+import uk.gov.hmcts.juror.api.moj.report.DataType;
+import uk.gov.hmcts.juror.api.moj.repository.PoolRequestRepository;
+import uk.gov.hmcts.juror.api.moj.repository.trial.TrialRepository;
+import uk.gov.hmcts.juror.api.moj.utils.SecurityUtil;
+
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class PanelMembersStatusReport extends AbstractStandardReport {
+
+    private final TrialRepository trialRepository;
+
+    @Autowired
+    public PanelMembersStatusReport(PoolRequestRepository poolRequestRepository, TrialRepository trialRepository) {
+        super(
+            poolRequestRepository,
+            QPanel.panel,
+            DataType.JUROR_NUMBER,
+            DataType.PANEL_STATUS
+        );
+
+        isCourtUserOnly();
+        this.trialRepository = trialRepository;
+    }
+
+    @Override
+    protected void preProcessQuery(JPAQuery<Tuple> query, StandardReportRequest request) {
+        query.where(QPanel.panel.trial.trialNumber.eq(request.getTrialNumber()));
+        query.where(QPanel.panel.trial.courtLocation.owner.eq(SecurityUtil.getLocCode()));
```

Comment:
Should be SecurityUtil.getOwner()

Reply:
updated.

Reply:
Following discussions with the business it was agreed that this should be locCode.eq(LocCode)